specification/batch/Azure.Batch/routes.tsp

```diff
@@ -0,0 +1,2237 @@
+#suppress "@azure-tools/typespec-azure-core/no-error-status-codes" "Azure REST API guidelines recommend using 'default' error response for all error cases. Avoid defining custom 4xx or 5xx error cases."
+#suppress "@azure-tools/typespec-azure-core/no-rpc-path-params" "Operations defined using BatchOperation should not have path parameters."
+#suppress "@azure-tools/typespec-azure-core/casing-style" "The names of Model types must use camelCase"
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Operation 'Exists' should be defined using a signature from the Azure.Core namespace."
+#suppress "@azure-tools/typespec-azure-core/rpc-operation-request-body" "BatchOperation with '@get' cannot have a body."
+interface Batch {
+  @summary("Lists all of the applications available in the specified Account.")
+  @doc("""
+This operation returns only Applications and versions that are available for
+use on Compute Nodes; that is, that can be used in an Package reference. For
+administrator information about applications and versions that are not yet
+available to Compute Nodes, use the Azure portal or the Azure Resource Manager
+API.
+""")
+  @route("/applications")
+  @projectedName("java", "listApplicationsInternal")
+  listApplications is Ops.ListOperation<{}, BatchApplicationListResult>;
+
+  @summary("Gets information about the specified Application.")
+  @doc("""
+This operation returns only Applications and versions that are available for
+use on Compute Nodes; that is, that can be used in an Package reference. For
+administrator information about Applications and versions that are not yet
+available to Compute Nodes, use the Azure portal or the Azure Resource Manager
+API.
+""")
+  @route("/applications/{applicationId}")
+  @projectedName("java", "getApplicationInternal")
+  getApplication is Ops.ReadOperation<
+    {
+      @doc("The ID of the Application")
+      @path
+      applicationId: string;
+    },
+    BatchApplication
+  >;
+
+  @summary("""
+Lists the usage metrics, aggregated by Pool across individual time intervals,
+for the specified Account.
+""")
+  @doc("""
+If you do not specify a $filter clause including a poolId, the response
+includes all Pools that existed in the Account in the time range of the
+returned aggregation intervals. If you do not specify a $filter clause
+including a startTime or endTime these filters default to the start and end
+times of the last aggregation interval currently available; that is, only the
+last aggregation interval is returned.
+""")
+  @route("/poolusagemetrics")
+  @projectedName("java", "listPoolUsageMetricsInternal")
+  listPoolUsageMetrics is Ops.ListOperation<
+    {
+      @doc("""
+    The earliest time from which to include metrics. This must be at least two and
+    a half hours before the current time. If not specified this defaults to the
+    start time of the last aggregation interval currently available.
+    """)
+      @query
+      starttime?: utcDateTime;
```

Comment:
SDK. Do we need to project to client `startTime`?

Reply:
updated